systemd/schema-registry.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Schema Registry
+Documentation=http://docs.confluent.io/current/schema-registry/docs/index.html
+Requires=network.target
+Wants=kafka-server.service
```

Comment:
The "Wants" is a systemd directive that tells this service will attempt to start the specified service (kafka-server.service) if it's activated.  If it's absent or inactivated, the current service will continue to start.  In other words, having a "wants" on other services is a nice-to-have but not required.
Note that network is required though (line 4).